app/views/organizations/home/index.html.erb

```diff
@@ -81,11 +81,11 @@
 
       <div class="d-flex flex-column justify-content-center align-items-center">
         <div class="mb-3">
-          <%= image_tag('paw-account.svg', width: '150', alt: "Create and account") %>
+          <%= image_tag('paw-account.svg', width: '150', alt: "Create an account") %>
         </div>
         <div>
           <p class="text-black">
-            1. Create Account
+            <%= t("general.step_1") %>
```

Comment:
What do you think about putting all of the steps in an ordered list? Then we won't need to add the step number to the translation. The translation key would be more descriptive `t(".create_account")`

Reply:
Will need to look at it again, but should be doable!